packages/extra/bootbox/templates/matchbox/deployment.yaml

```
@@ -0,0 +1,54 @@
+    matchLabels:
+      app: bootbox-matchbox
+  template:
+    metadata:
+      labels:
+        app: bootbox-matchbox
+    spec:
+      containers:
+      - name: matchbox
+        image: "{{ $.Files.Get "images/matchbox.tag" | trim }}"
+        args:
+          - "-address=:8080"
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)